### PR TITLE
Update configuration files to match latest version

### PR DESCRIPTION
**Pull request by RyanCheungJF**
Resolves [#1768](https://github.com/reposense/RepoSense/issues/1768).

Based changes off the latest commit on the `author-config.csv` in the main repo as shown [here](https://github.com/reposense/RepoSense/blob/master/config/author-config.csv). Mainly, two fields are changed.

```
Currently, there is a warning due to the outdated header fields in both 
the author-config and repo-config CSV files.

Let's update both CSV files so that the headers are now synchronized
with what is supported in the latest version of RepoSense.
```

### CODE REVIEW

**Comment by dcshzj** [comment]
Hey! Thanks for the PR! Are you also able to update the `repo-config.csv` file as well? There are [a few new columns that were added over time](https://reposense.org/RepoSense/ug/configFiles.html#repo-config-csv) but the file in this repository was not updated.

**Comment by RyanCheungJF** [comment]
Sure @dcshzj , I've updated my PR to update the other config file. Do let me know if you need anything else updated. Thanks in advance.

File: configs/repo-config.csv [hunk 1]
```diff
@@ -1,2 +1,2 @@
-Repository's Location,Branch,File formats,Ignore Glob List,Ignore standalone config,Ignore Commits List,Ignore Authors List
-https://github.com/reposense/RepoSense.git,master,,,,,
+Repository's Location,Branch,File formats,Ignore Glob List,Ignore standalone config,Ignore Commits List,Ignore Authors List,Shallow Cloning,Find Previous Authors
```
Comment (dcshzj) [inline configs/repo-config.csv:1]: It looks like a number of columns are still missing, do refer to the documentation on the full list of columns that should be present:
1. File Size Limit
2. Ignore File Size Limit
3. Skip Ignored File Analysis

If possible, do also follow the order that is presented in the user guide so it is easier for our users to follow.

**Review by dcshzj — CHANGES_REQUESTED** [review]

File: configs/repo-config.csv (hunk 1 above)
Comment (RyanCheungJF) [reply configs/repo-config.csv]: Ah, my bad, I was following the config file in the reposense repo. I've updated and ordered them according the the docs. Let me know if you need any more changes.

Comment (dcshzj) [reply configs/repo-config.csv]: @RyanCheungJF The newline needs to be added as well, currently the `repo-config.csv` file is missing a final newline.

**Review by dcshzj — APPROVED** [review]
LGTM!

**Pull request merged**